### ISSUE TITLE
Fullscreen mode is overly always-on-top (windows)

### ISSUE THREAD

**Issue by vith** [u8sand/Baka-MPlayer#237]
Not sure if this is partially intentional, but it causes some issues.

When you alt-tab, the player no longer has focus but stays on top of everything, including the alt-tab UI. So at that point you can no longer exit fullscreen or tell which window actually has focus.

The only reliable way out of this state I found (other than getting lucky alt-tabbing back to the player) is to Ctrl+Alt+Del. For some reason focus always returns to the player when returning out of that special menu.

Similar problems occur when opening the start menu with the Windows key. You can't see it, but it has focus and is accepting input, and the player is not.

OS: Windows 10
Baka-MPlayer: 2.0.4

**Comment by vith** [comment]
I should mention I have the `On Top` preference set to `Never`.

**Comment by u8sand** [comment]
So if I understand correctly: Alt-tab from Full screen mode causes this issue? Does this occur only when the video continues to be played in fullscreen mode?

**Comment by vith** [comment]
Yes, it only happens in fullscreen mode and the video keeps playing the whole time.

1. start playing video
2. enter fullscreen with keybind
3. alt-tab (can't see the alt-tab UI but it still changes window focus)
4. video is still fullscreen and still playing
5. try to exit fullscreen mode with any of {same keybind, escape key, double click}, doesn't work because player doesn't have focus

**Comment by u8sand** [comment]
I'm under the impression this might be caused by mpv not paying attention to our window's focus. I imagine  when we move to a shared gl context with mpv this will be resolved. I hate to say that this won't happen today, but rest assured we're working on it.
